app/src/main/res/values/strings.xml

```diff
@@ -533,36 +533,31 @@
-        <item>Can I extract images or text from a PDF?#####Yes, to extract images or text from a PDF, use Extract Images and Extract Text available on the home page or in the options menu which appears if you swipe from left to right or click on the 3 horizontal lines at the top left of the app.</item>
-        <item>Before converting a text file into a PDF, can I specify enhancement options?#####Yes, you can use the enhancement options available in the Text to PDF option.</item>
-        <item>Can I convert a PDF into different file formats such as PPT, PPTX, ODT, DOC, DOCX?#####No, the PDF can only be converted to text format at the moment but we are planning to add other conversion options soon!</item>
-        <item>Is Images-to-pdf software available to any other operating system?#####There is no intention to migrate this software to another operating system.</item>
-        <item>Where can I download Images-to-pdf software?#####The software can be downloaded from Google PlayStore and also on F-droid</item>
-        <item>Can I contribute to Images-to-pdf project?#####Yes, please! \nFeel free to share your ideas(feature requests), submit a bug or pull request for anything you feel should be added. \nThis project currently has multiple contributors that add as little as a single line change. This project is opened to contributions of any kind. You can also fork the project and further develop it on your own so long as you work within the confines of the license. Pull requests of your contributions will be merged if they fulfill some certain requirements. Read the project\'s Contributor\'s guide to get started on how to contribute.</item>
-        <item>I have some questions not contained in this FAQs document, how can I contact the developer?#####For further inquiries, you can contact the developer via email. Send an email to swati4star@gmail.com.</item>
+        <item>What image file formats are supported?#####The app supports jpeg, jpg, tiff, gif, psd, bmp, eps, png, etc.</item>
+        <item>Can I create a new text file using the app?#####No, you cannot. You can only convert an already created text file to PDF.</item>
+        <item>Is there a limit on number of images I can add at once to create a PDF?#####No, there are no restrictions.</item>
+        <item>Is there a limit to the image size I can add?#####No, there are no limits. However, we advise to restrict the size to less than 10MB. The conversion of multiple large image files take upwards of 10 seconds.</item>
+        <item>Can I insert captions in the image after adding?#####No, you cannot.</item>
+        <item>Can I read PDF files using the app?#####No, you cannot as of now. This app does not work as a PDF viewer.</item>
```

Comment:
The previous answer was fine 
`Yes, you can open pdf files with the software but you need an external pdf reader to be able to view its content.`

Reply:
This can confuse readers. Even if users open the PDF file using the app they cannot read it. 

The question is whether the app can be used to read PDF files. It's a no.